docs/reference/commandline/node_ls.md

```diff
@@ -161,6 +166,11 @@ e216jshn25ckzbvmwlnh5jr3g: swarm-manager1 Ready
 35o6tiywb700jesrt3dmllaza: swarm-worker1 Needs Rotation
 ```
 
+To print all the nodes with json output, use `json` directive:
```

Comment:
Can you add a blank line between the text and the code block? Also perhaps;

```suggestion
To print all nodes in JSON format, use the `json` directive:

```

Reply:
Perhaps also for the other docs that were added (JSON uppercase, "_the_ `json` directive"